cpp/src/arrow/compute/api_aggregate.cc

```diff
@@ -41,8 +41,12 @@ Result<Datum> MinMax(const Datum& value, const MinMaxOptions& options, ExecConte
   return CallFunction("min_max", {value}, &options, ctx);
 }
 
-Result<Datum> Mode(const Datum& value, const ModeOptions& options, ExecContext* ctx) {
-  return CallFunction("mode", {value}, &options, ctx);
+Result<Datum> Any(const Datum& value, ExecContext* ctx) {
+  return CallFunction("any", {value}, ctx);
+}
+
+Result<Datum> Mode(const Datum& value, ExecContext* ctx) {
```

Comment:
The ModeOptions are dropped here, might be a rebase error?

Reply:
Thanks - yes, that's the problem!